app/presenters/indexable_presenter.rb

```diff
@@ -0,0 +1,25 @@
+class IndexablePresenter
+
+  def initialize(policy)
+    @policy = policy
+  end
+
+  def indexable_attributes
+    {
+      title: policy.name,
+      description: policy.description,
+      link: id,
+      indexable_content: "",
```

Comment:
I'd prefer indexable content to be blank if it doesn't have any novel content: the text from the description field will be included in searches anyway.